src/axom/core/utilities/Timer.hpp

```diff
@@ -160,6 +193,8 @@ class Timer
   TimeStruct m_startTime;
   TimeStruct m_stopTime;
   bool m_running;
+  TimeDiff m_summedTime;
```

Comment:
Do you think people might prefer an array of the timings, which they could use for stats, e.g. to compute the mean and variance of the runtimes? Or the average of the N most recent times?

Reply:
Well, I'm the only user right now, and I just need the average.  Users who need more can certainly store an array of time values themselves and compute moments on those values.  I'm a bit concerned about any implementation that may have to allocate data in the middle of timing, since that could add excessive or unpredictable overheads.